app/models/post.rb

```diff
@@ -0,0 +1,21 @@
+class Post < ApplicationRecord
+  belongs_to :user, foreign_key: 'AuthorId'
```

Comment:
- [ ] The Author model is an alias to the User model, as such your association should be 
`belongs_to :author, class_name: 'User'`